README.md

```diff
@@ -152,21 +172,25 @@ The "Secure Login" button on your website/app opens the native SecureLogin app:
 At the same time it sends a request to your `/login` action:
 
 ```javascript
-loginaccount.onclick=function(){
-  xhr('/login',{
-    sltoken: SecureLogin(), //returns state and opens the app
-    authenticity_token: csrf
-  }, function(d){
-    if(d == 'ok'){
-      // force focus, useful for Chrome in full screen
-      //if(document.visibilityState!='visible') alert("Logged in successfully.")
-      location.reload()
-    }else{
-      console.log(d)
+loginAccount.addEventListener('click', function() {
+  xhr(
+    '/login',
+    {
+      sltoken: SecureLogin(), //returns state and opens the app
+      authenticity_token: csrf
+    },
```

Comment:
I'm all for generous spacing, but this is taking it too far IMO.
README should be concise.